### DISCUSSION TITLE
Calibre

### DISCUSSION THREAD

**Discussion by SakethKanchi** [catppuccin/catppuccin#1854] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Maybe

**Additional context 📷 (optional)**
Calibre E-book Viewer. It's mostly CSS as far as I know.

![image](https://user-images.githubusercontent.com/70198692/152743305-8c03adad-bd99-40e7-9b2f-5202e2a63e49.png)

**Comment by 112cxyz** [comment]
Although this hasn't been touched in many months ill try to port it!

**Reply by Starcr0ss** [reply]
Just curious, how is the port going?

**Reply by romanskorobagatko** [reply]
How is it going?

**Reply by 112cxyz** [reply]
I tried doing it but custom css was completely broken and I am busy with work now. I do urge you if you want it you can try to port it.

**Comment by 42willow** [comment]
I'll do this in two weeks when I get back from holidays

I call dibs :)

**Comment by 112cxyz** [comment]
Give it your best shot,
I had issues with getting it to work on all platforms, Linux would just
stick to the default QT theme and macOS used it’s basic theme even when I
had the custom theme applied.

**Reply by 42willow** [reply]
:eyes: looks like there's a custom theme section

![image](https://github.com/catppuccin/catppuccin/assets/70784752/faa37dd0-618f-41fd-909d-eca524b23e6e)

I'll probably just use whiskers to generate the file for this


I'm on 7.7.0

**Reply by 112cxyz** [reply]
Holy moly, that was not there when I did it.

Although I haven't attempted to do it since early last year

**Comment by SakethKanchi** [comment]
> 👀 looks like there's a custom theme section
> 
> ![image](https://private-user-images.githubusercontent.com/70784752/317995980-faa37dd0-618f-41fd-909d-eca524b23e6e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3MDYwNjYsIm5iZiI6MTcxMTcwNTc2NiwicGF0aCI6Ii83MDc4NDc1Mi8zMTc5OTU5ODAtZmFhMzdkZDAtNjE4Zi00MWZkLTkwOWQtZWNhNTI0YjIzZTZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI5VDA5NDkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThhMTEzYTMxNjg2ZTk3MmZjMGI3ZWI0MmI0NTQxNTIwNGExYWUwYzdhNDRlNWI5ZTE4YjYxMjM1NDg2ZGIxMDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UsZyM1CvelCvMPSnspRc8CnEkfrG7Cv5YpeMtRk5fDA)
> 
> I'll probably just use whiskers to generate the file for this
> 
> I'm on 7.7.0

These are just the colors for the application UI. The reader UI colors still remain the same.

**Reply by sphars** [reply]
Setting to custom colors affected my reader UI colors:

![Screenshot_20240402_214927](https://github.com/catppuccin/catppuccin/assets/18315081/420021c9-38ec-41ca-943d-00fbe2abffe7)

**Comment by sphars** [comment]
Don't know if it helps, but when using the Catppuccin themes in KDE, I was able to import the colors using the 'Use system colors' option. Just had to change two fields that were set as #000000. See my image above. Here's the values it created (my KDE theme is Macchiato): [catpuccin-macchiato.calibre-palette.json](https://github.com/catppuccin/catppuccin/files/14846429/catpuccin-macchiato.calibre-palette.json)

**Comment by 42willow** [comment]
https://github.com/catppuccin/catppuccin/issues/2367

https://github.com/42Willow/calibre

hopefully the colours are right :/ although it's not hard to change

**Comment by sgoudham** [comment]
Closing as #2367 exists
